udaykalra/walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/style.css

```diff
@@ -7,3 +7,7 @@
 #quote-container {
   margin-top: 20px;
 }
+
+body {
+    background: white;
+}
```

Comment:
Adding a newline at the end of file would avoid introducing this warning.

Reply:
This is an example demo, not the webpage.

Reply:
Should I still make changes and re-review?